### PR TITLE
added toggle button for database selection

### PR DESCRIPTION
**Pull request by aniarya82**
screen shot 2018-05-29 at 15 53 53
Note: Currently if a single database is selected and then the user clicks toggle all, then the selected database gets unselected and the ones not selected gets selected

### CODE REVIEW

File: public/js/search.js [hunk 1]
```diff
@@ -491,6 +491,24 @@ var Databases = React.createClass({
         this.setState({type: type});
     },
 
+    handleToggle: function (type) {
```
Comment (yeban) [inline public/js/search.js]: A much simpler way exists:
```
$(`.${type} .database`).click();
```

File: public/js/search.js [hunk 2]
```diff
@@ -504,7 +522,18 @@ var Databases = React.createClass({
                                     className="panel panel-default">
                                     <div
                                         className="panel-heading">
-                                        <h4>{category[0].toUpperCase() + category.substring(1).toLowerCase() + " databases"}</h4>
+                                        <h4 style={{display: "inline"}}>{category[0].toUpperCase() + category.substring(1).toLowerCase() + " databases"}</h4>
+                                        <p className="" style={{display: "inline"}}>
```
Comment (yeban) [inline public/js/search.js]: Sorry, I think left aligned with the header will be better:

screen shot 2018-05-30 at 12 08 31

```
<div
    className="panel-heading">
    <h4 style={{display: "inline"}}>{category[0].toUpperCase() + category.substring(1).toLowerCase() + " databases"}</h4>
    &nbsp;&nbsp;
    <a disabled={true}
        style={{cursor: "pointer"}}
        onClick={
        _.bind(function () {
        this.handleToggle(category)
        }, this)
        }>
        [Toggle all]
    </a>
</div>
```

File: public/js/search.js (hunk 1 above)
Comment (yeban) [reply public/js/search.js]: Further,

1. When nucleotide databases are selected, 'Toggle all' button for protein databases should be disabed and vice versa.
2. 'Toggle all' button should only be shown when there are two or more databases.

**Comment by yannickwurm** [comment]
I'll add that many of us don't know what "toggle" means. So best to use Select and Deselect All 

( I had a moment of doubt, we want [deselect not unselect](https://english.stackexchange.com/questions/18465/unselect-or-deselect#18466) )

**Pull request merged**